vignettes/dark_ink_transition/lib/dark_ink_bar.dart

```diff
@@ -3,40 +3,38 @@ import 'package:flutter/material.dart';
 import 'main.dart';
 
 class DarkInkBar extends StatefulWidget {
+  final ValueNotifier<bool>? darkModeValue;
 
-  final ValueNotifier<bool> darkModeValue;
-
-  DarkInkBar({ this.darkModeValue });
+  DarkInkBar({this.darkModeValue});
 
   @override
   State createState() {
-    return _DarkInkBarState(darkModeValue);
+    return _DarkInkBarState(darkModeValue!);
```

Comment:
This is quite dangerous, someone could pass in null and cause a RTE error.

Instead just mark `darkModeValue` as `required`